gcc/rust/resolve/rust-toplevel-name-resolver-2.0.cc

```diff
@@ -168,15 +169,15 @@ TopLevel::visit (AST::ExternCrate &crate)
 		crate.get_referenced_crate ());
 }
 
-static bool
-is_macro_export (AST::MacroRulesDefinition &def)
-{
-  for (const auto &attr : def.get_outer_attrs ())
-    if (attr.get_path ().as_string () == Values::Attributes::MACRO_EXPORT)
-      return true;
+// static bool
```

Comment:
You need to remove this comment since you have moved the function over to another file